pkg/jwt/jwt.go

```diff
@@ -0,0 +1,68 @@
+package jwt
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"github.com/MaxBoych/gofermart/pkg/logger"
+	"github.com/golang-jwt/jwt/v5"
+	"go.uber.org/zap"
+	"time"
+)
+
+func GenerateSecretKey() (string, error) {
+	key := make([]byte, 32)
+	if _, err := rand.Read(key); err != nil {
+		logger.Log.Error("Error to generate secret key", zap.Error(err))
+		return "", err
+	}
+	return base64.StdEncoding.EncodeToString(key), nil
+}
+
+func GenerateTokenValue(userID int64, key string) (string, error) {
+	claims := jwt.MapClaims{
+		"user_id":    userID,
+		"expiration": time.Now().Add(time.Hour * 72).Unix(),
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	signedToken, err := token.SignedString([]byte(key))
+	if err != nil {
+		logger.Log.Error("Error to generate jwt", zap.Error(err))
+		return "", err
+	}
+
+	return signedToken, nil
+}
+
+func ValidateTokenAndGetUserID(tokenValue string, key string) (int64, error) {
+	token, err := jwt.Parse(tokenValue, func(token *jwt.Token) (interface{}, error) {
```

Comment:
```suggestion
	token, err := jwt.Parse(tokenValue, func(_ *jwt.Token) (interface{}, error) {
```
не пользеушься параметром - не пользуйся, зачем копировать значение в функцию?